content/pandas/concepts/dataframe/terms/at/at.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: '.at[]'
+Description: 'Returns a specific value in row/column pairs in a dataset.' 
+Subjects: 
+  - 'Computer Science'
+  - 'Data Science'
+Tags: 
+  - 'Data Structures'
+  - 'Pandas'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+In Pandas, the **`.at[]`** function is used as an accessor to fetch a specific value from a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) using row/column pairs.
```

Comment:
``` suggestion
In Pandas, the **`.at[]`** function is used as an accessor to fetch a specific value from a [`DataFrame`](https://www.codecademy.com/resources/docs/pandas/dataframe) using row and column pairs.
```
Made a slight change for `row/column pairs`. In pandas for the `.at[ ]` function fetch using both row and column pairs, the use of the `/` symbol shows that the `.at[ ]` function either uses rows or columns to fetch a specific value.